### PR TITLE
chore: Changelog + Release Notes for v2.0.0-rc1

### PR DESCRIPTION
**Pull request by clevinson**
## Description

This PR updates changelog & adds release notes for v2.0.0-rc1.

After PR is merged the following actions should take place:
- [ ] Tag v2.0.0-rc1 off of the commit (aka the HEAD of `release/v2.0.x`)
- [ ] Merge changelog updates from this PR back into master

**Note:** Whatever ends up being the last RC will have one final commit after it that tags versions of all the necessary submodules, along with final publishing of the v2.0.0 release.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [x] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [x] updated the relevant documentation or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking.
+
+For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
+
+### Ecocredit Module
+
+The ecocredit module enables the ability to manage classes of ecosystem service credits and to mint credits through a batch issuance process. For more information about the ecocredit module, see the specification:
+
+- [Ecocredit Module Specification](https://docs.regen.network/modules/ecocredit/)
```
Comment (technicallyty) [inline RELEASE_NOTES.md:34]:
```suggestion
The ecocredit module enables the ability to manage classes of ecosystem service credits and to mint credits through a batch issuance process. For more information about the ecocredit module, see the [Ecocredit Module Specification](https://docs.regen.network/modules/ecocredit/).


```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+The authz module enables a granter to grant an authorization to a grantee that allows the grantee to execute a message on behalf of the granter. For more information about the authz module, see the specification:
+
+- [Authz Module Specification](https://docs.cosmos.network/master/modules/authz/)
+
+### Feegrant Module
+
+The feegrant module enables the ability for a granter to grant an allowance to a grantee where the allowance is used to cover fees for sending transactions. For more information about the feegrant module, see the specification:
+
+- [Feegrant Module Specification](https://docs.cosmos.network/master/modules/feegrant/)
```
Comment (technicallyty) [inline RELEASE_NOTES.md:22]:
```suggestion
The feegrant module enables the ability for a granter to grant an allowance to a grantee where the allowance is used to cover fees for sending transactions. For more information about the feegrant module, see the [Feegrant Module Specification](https://docs.cosmos.network/master/modules/feegrant/).
```

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+For more information about Cosmos SDK v0.43 and Cosmos SDK v0.44, see the release notes:
+
+- [Cosmos SDK v0.43.0 Release Notes](https://github.com/cosmos/cosmos-sdk/blob/release/v0.43.x/RELEASE_NOTES.md)
+- [Cosmos SDK v0.44.0 Release Notes](https://github.com/cosmos/cosmos-sdk/blob/release/v0.44.x/RELEASE_NOTES.md)
+
+### Authz Module
+
+The authz module enables a granter to grant an authorization to a grantee that allows the grantee to execute a message on behalf of the granter. For more information about the authz module, see the specification:
+
+- [Authz Module Specification](https://docs.cosmos.network/master/modules/authz/)
```
Comment (technicallyty) [inline RELEASE_NOTES.md:16]:
```suggestion
The authz module enables a granter to grant an authorization to a grantee that allows the grantee to execute a message on behalf of the granter. For more information about the authz module, see the [Authz Module Specification](https://docs.cosmos.network/master/modules/authz/).
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+### Feegrant Module
+
+The feegrant module enables the ability for a granter to grant an allowance to a grantee where the allowance is used to cover fees for sending transactions. For more information about the feegrant module, see the specification:
+
+- [Feegrant Module Specification](https://docs.cosmos.network/master/modules/feegrant/)
+
+### Permanent Locked Accounts
+
+Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking.
```
Comment (ryanchristo) [inline RELEASE_NOTES.md:26]:
```suggestion
Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` message. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking.
```

File: RELEASE_NOTES.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
```
Comment (technicallyty) [inline RELEASE_NOTES.md:28]:
```suggestion
Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking. For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
```

**Review by technicallyty — APPROVED** [review]
lgtm, just a few nits on putting links inline 👍🏻

File: RELEASE_NOTES.md (hunk 4 above)
Comment (ryanchristo) [reply RELEASE_NOTES.md:26]: Ok to leave as is but the CLI uses the message and it might be nice to spell it out when not using backticks.

**Review by ryanchristo — APPROVED** [review]
Looks good to me. The changelog excludes a lot of the minor audit changes but I think all the necessary changes are included.

File: RELEASE_NOTES.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+The authz module enables a granter to grant an authorization to a grantee that allows the grantee to execute a message on behalf of the granter. For more information about the authz module, see the [Authz Module Specification](https://docs.cosmos.network/master/modules/authz/).
+
+### Feegrant Module
+
+The feegrant module enables the ability for a granter to grant an allowance to a grantee where the allowance is used to cover fees for sending transactions. For more information about the feegrant module, see the [Feegrant Module Specification](https://docs.cosmos.network/master/modules/feegrant/).
+
+### Permanent Locked Accounts
+
+Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` Msg and via CLI. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking. For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
```
Comment (clevinson) [inline RELEASE_NOTES.md:22]:
```suggestion
Regen Ledger v2.0 supports the on-chain creation of permanent locked accounts through the `MsgCreatePermanetLockedAccount` message. These special types of accounts are intended to be used by Regen Foundation for seeding Community Staking DAOs, wherein the initial REGEN funds distributed to these accounts must be permanently locked and only usable for governance and staking. For more information see [regen-ledger#188](https://github.com/regen-network/regen-ledger/issues/188)
```

**Comment by clevinson** [comment]
> Looks good to me. The changelog excludes a lot of the minor audit changes but I think all the necessary changes are included.

@ryanchristo yeah.. I tried to include all the ones that I thought made sense. If there's any in particular you would like to see included pls lmk!

**Pull request merged**